.github/workflows/needs-attention.yml

```diff
@@ -0,0 +1,21 @@
+name: Issue Needs Attention
+# This workflow is triggered on issue comments.
+on:
+  issue_comment:
+    types: created
+
+jobs:
+  applyNeedsAttentionLabel:
+    name: Apply Needs Attention Label
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Apply Needs Attention Label
+        uses: hramos/needs-attention@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          response-required-label: "Needs: Author Feedback"
+          needs-attention-label: "Needs: Attention"
+        id: needs-attention
```

Comment:
What is the id for?

Reply:
Assigning an id to a step lets us refer back to it later. In this case, it's used in L21 to get the output of this step via `steps.needs-attention`.